edge/discussion_materials/ONNX Edge scope and profile definition.md

```diff
@@ -0,0 +1,76 @@
+# ONNX Edge scope and profile definition
+Author: Yedong Liu, Milan Oljaca, Saurabh Tangri, Ofer Rosenberg
+
+## Motivation
+
+The following document provides a description of terminology related to "edge" : what is the scope of "edge", what are the scenarios, example to devices which map into the edge, and more. The goal is to define what is contained inside an edge profile , based on attributes.
```

Comment:
- Example of devices 
- Consider rewording "The goal is to define what is contained inside an edge profile , based on attributes." to "The goal is to define elements that identify an edge profile,"